austation/code/modules/surgery/organs/tongue.dm

```diff
@@ -0,0 +1,77 @@
+/obj/item/organ/tongue
+	var/static/list/austation_languages_possible_base = typecacheof(list(
+		/datum/language/buzzwords,
+		/datum/language/moffic,
+		/datum/language/sylvan,
+		/datum/language/shadowtongue,
+		/datum/language/terrum,
+	))
+
+/obj/item/organ/tongue/fly
+	var/static/list/languages_possible_fly = typecacheof(list(
+		/datum/language/common,
+		/datum/language/draconic,
+		/datum/language/codespeak,
+		/datum/language/monkey,
+		/datum/language/narsie,
+		/datum/language/beachbum,
+		/datum/language/aphasia,
+		/datum/language/piratespeak,
+		/datum/language/moffic,
+		/datum/language/sylvan,
+		/datum/language/shadowtongue,
+		/datum/language/terrum,
+		/datum/language/buzzwords
+	))
+
+/obj/item/organ/tongue/fly/Initialize(mapload)
+	. = ..()
+	languages_possible = languages_possible_fly
+
+/obj/item/organ/tongue/bone
+	var/static/list/languages_possible_skeleton = typecacheof(list(
+		/datum/language/common,
+		/datum/language/draconic,
+		/datum/language/codespeak,
+		/datum/language/monkey,
+		/datum/language/narsie,
+		/datum/language/beachbum,
+		/datum/language/aphasia,
+		/datum/language/piratespeak,
+		/datum/language/moffic,
+		/datum/language/sylvan,
+		/datum/language/shadowtongue,
+		/datum/language/terrum,
+		/datum/language/calcic
+	))
+
+/obj/item/organ/tongue/bone/Initialize()
+	. = ..()
+	languages_possible = languages_possible_skeleton
+
+/obj/item/organ/tongue/ethereal
+	name = "electric discharger"
+	desc = "A sophisticated ethereal organ, capable of synthesising speech via electrical discharge."
+	icon_state = "electrotongue"
```

Comment:
this isn't modularized

Reply:
need to modularize surgery.dmi too. .dmi files are really really bad for conflicts, so it's important we don't change the core files for them

Reply:
piss